CHANGELOG.md

```diff
@@ -4,6 +4,45 @@ All notable changes to this project will be documented in this file. And this pr
 
 ## [Unreleased]
 
+### Framework
+
+- [Optimization] Bump Rust toolchain to `1.34.2`. [@kaikai1024]
+- [Optimization] Use Rust 2018 edition. [@kaikai1024]
+- [Optimization] Use `cita-logger` crate. [@kaikai1024]
+
+### Network
+
+- [Fix] Fix the bug that save wrong session. [@leeyr338]
+- [Fix] Should not unwrap on handle_remote_msg. [@leeyr338]
+- [Fix] Fix the operation of repeated peer key. [@jerry-yu]
+
+### System Contracts
+
+- [Fix] Fix the import path. [@ouwenkg]
+- [Optimization] Use `protocol version` instead of version. [@kaikai1024]
+
+### Tools
+
+- [Refactor] Rewrite the tool of creating genesis using Rust. [@ouwenkg]
+
+### Test
+
+- [Refactor] Refactor scripts of integrate test. [@kaikai1024]
+
+### Scripts
+
+- [Fix] Node path should not consider bin path. [@rainchen]
+- [Refactor] Refactor the `cita.sh` script. [@kaikai1024]
+- [Fix] Fix amend help info and latest version. [@leeyr338]
+- [Optimization] Rename `config_example` to `default_config`. [@kaikai1024]
+
```

Comment:
* recover security audit test
* upgrade protobuf

Reply:
wait a minute.

Reply:
My fault. Just ignore this item.